optimisation/ScanSelectionsTree.py

```diff
@@ -326,7 +330,9 @@
 
             expBkg *= nExpEv / bkgConfig['nEvents'] / bkgConfig['fractiontokeep']
 
-            # S/B and significance and Signal Error
+            expBkg *= hBkgCorrFactor.GetBinContent(hBkgCorrFactor.FindBin((ptMax+ptMin)/2))
```

Comment:
If I am not wrong here you need an `if` because this has to be executed only in case one enables the correction factor from the config file. Is it correct?

Reply:
You are probably right, I was not aware of this because I usually use the correction factor anyway. 
However, I'll fix it tomorrow. 
Thank you!